content/800-data-platform/200-projects/120-import-existing-project-existing-db.mdx

```diff
@@ -0,0 +1,79 @@
+</TopBlock>
+
+## Prerequisites and considerations
+
+* GitHub account
+* GitHub repository with your Node.js application that uses Prisma ORM
+* Connection string to a [Prisma ORM-supported databases](/reference/database-reference/supported-databases)
+* [Sign up for the Prisma Data Platform](/data-platform/accounts#sign-up-for-the-prisma-data-platform)
+* [Grant the Prisma GitHub app read and write permissions to your GitHub repositories](/data-platform/projects#grant-read-and-write-permissions-to-the-prisma-github-app)
+
+## Steps
+
+1. Go to the [_Projects_](https://cloud.prisma.io/projects) page and click **New Project**.
+2. On the _Configure project_ page, name your project and select an existing GitHub repository.
+    1. In the **Display Name** text box, provide a name for your project.
+        <Admonition>
+        
+        **Note**<br /><br />
+        The display name of a project appears in several places in the Prisma Data Platform.<br />
+          &nbsp;• [_Projects_](https://cloud.prisma.io/projects) page<br />
+          &nbsp;• [_Billing_](https://cloud.prisma.io/billing) page<br />
+          &nbsp;• **Projects** drop-down in the upper-left
+        </Admonition>
+    2. From the **GitHub Accounts & Organizations** drop-down menu, select a GitHub account or an organization.
+    3. Select the **Import a Prisma repository** option.
+    4. From the **Repository** drop-down menu, select the GitHub repository you want to import.
+    5. From the **Branch** drop-down menu, select the branch you want to import.
+    6. In the Prisma Schema path, check that the import wizard detects the path of the `schema.prisma` file correctly.
+    7. Click **Next**.
+        <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666314907/docs/pdp/create-manage-projects/import-project-01-configure-project_lgom2u.png"
+        alt="Prisma Data Platform - Manage project members - Step 03 - Click Edit for the project member you want to delete"
+        width="750px"
+        />
+3. On the _Configure environment_ page, connect your database and configure the Data Proxy location.
+    1. In the **Connection string** text box, paste your database connection string.
+    2. Under _Data Proxy_, from the **Location** drop-down menu, select the geographic location for the Data Proxy.
+        <Admonition>
+        
+        **Note**<br /><br />
+        Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+        If you select a different location, when your application sends requests to your database through the Data Proxy, the responses might be slow or might time out due to higher latency.
+        </Admonition>
+    3. From the **Static IPs** drop-down menu, select **Enabled** if your database is behind a firewall and you can only configure external access from specific IP addresses. Copy the IP addresses and add them to the allowlist of your database.
```

Comment:
```suggestion
    3. From the **Static IPs** drop-down menu, select **Enabled** if your database is behind a firewall and you can only configure external access from specific IP addresses. Copy the IP addresses and add them to your database's allowlist.
```

Makes it slightly briefer. There's a similar line in the **create new application** page, so if you make this change here, don't forget to make it there as well. Sorry that I missed it in my review of that page!

Reply:
Ah OK - I wasn't aware of this (I might have been aware of it at some point and forgotten it)